martin/martin-ui/src/components/dialogs/sprite-download.tsx

```diff
@@ -94,9 +95,12 @@ export function SpriteDownloadDialog({ name, sprite, onCloseAction }: SpriteDown
   return (
     <Dialog onOpenChange={(v: boolean) => !v && onCloseAction()} open={true}>
       <DialogContent className="max-w-2xl w-full max-h-[90vh] overflow-auto">
-        <DialogHeader>
+        <DialogHeader className="truncate">
           <DialogTitle className="text-2xl">
-            Download <code className="font-mono">{name}</code>
+            Download{' '}
+            <code className="font-mono">
+              <TooltipCopyText side="bottom" text={name} />
+            </code>
```

Comment:
this is already one level down ^^

```suggestion
            <TooltipCopyText side="bottom" text={name} />
```

Reply:
This was a format auto-correct, but are you saying just remove the `<code>` tag here?